proxy/rpc/np_gas_price.py

```diff
@@ -104,6 +112,26 @@ def model_post_init(self, _ctx) -> None:
             raise ValueError("'to', 'nonce' and 'gas' properties require the 'from'-property")
 
 
+class _RpcFeeHistoryResp(BaseJsonRpcModel):
+    baseFeePerGas: list[HexUIntField]
+    gasUsedRatio: list[float]
```

Comment:
Are there any requirements for precision in Ethereum protocol?

Reply:
not that I'm aware of. 
Also, it seems, Infura and Alchemy return arbitrary precision for floats.